server/routes/applications.js

```diff
@@ -96,14 +96,16 @@ router.get('/manage-applications/:appId/view/:env', (req, res, next) => {
   });
 });
 
-router.get('/manage-applications/:appId/update/:env', (req, res) => {
+router.get('/manage-applications/:appId/update/:env/:confirm?', (req, res) => {
```

Comment:
Is the `?` needed?

Reply:
It was the only way we found to make that an optional parameter as Moses wanted both handled in the same handler mapping.

Reply:
We wanted to keep the /confirm optional it only hits /confirm once we click the delete app button to then display `Are you sure you wish to delete....` part. Otherwise it will just display the update application page as shown on the prototype.

Reply:
Ok cool, thanks both for the clarification.